src/components/NetworkSwitcherPopover/index.tsx

```diff
@@ -34,7 +34,7 @@ export default function NetworkSwitcherPopover({ children, modal, placement }: N
   }
 
   function onNetworkChange(chainId: ChainId) {
```

Comment:
This check doesn't make sense anymore. Returning `selectNetwork(chainId)` will have same effect. I think cleaning can go even further - since `onNetworkChange` function isn't doing nothing meaningful, it's just a wrapper over `selectNetwork`, it can be entirely deleted. In line 42 just pass `onNetworkChange: selectNetwork` and we saved 3 lines of code \o/